src/Internal/Proxy/HooksProxy.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Class HooksProxy
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Proxy;
+
+/**
+ * Actions Proxy
```

Comment:
```suggestion
 * Hooks Proxy
```

Reply:
Fixed in Fixed in https://github.com/Automattic/woocommerce-payments/pull/6908/commits/c57fdca57b9394a3ea564c475aee875b7b32d4ba.